themes/demo/html/__header.html

```diff
@@ -1,5 +1,5 @@
-<div class="h2" style="background-color: purple; padding: 2px 2px 2px 16.666667%; color: white; text-align: center;">
-Demo
-</div>
+<script>
```

Comment:
The <div id="outer"> comes in the include at the bottom of this file, so don't think this'll work?